--- a/node_modules/node-llama-cpp/dist/config.js	2026-04-06 12:34:15.778925136 +0200
+++ b/node_modules/node-llama-cpp/dist/config.js	2026-04-06 12:34:19.315180468 +0200
@@ -8,26 +8,27 @@
 import { nodeLlamaCppGpuOptions, LlamaLogLevel, LlamaLogLevelValues, parseNodeLlamaCppGpuOption, nodeLlamaCppGpuOffStringOptions } from "./bindings/types.js";
 const __dirname = path.dirname(fileURLToPath(import.meta.url));
 const env = envVar.from(process.env);
+const nixCacheDir = path.join(os.homedir(), ".cache", "node-llama-cpp");
 export const llamaDirectory = path.join(__dirname, "..", "llama");
-export const llamaToolchainsDirectory = path.join(llamaDirectory, "toolchains");
+export const llamaToolchainsDirectory = path.join(nixCacheDir, "toolchains");
 export const llamaPrebuiltBinsDirectory = path.join(__dirname, "..", "bins");
-export const llamaLocalBuildBinsDirectory = path.join(llamaDirectory, "localBuilds");
+export const llamaLocalBuildBinsDirectory = path.join(nixCacheDir, "localBuilds");
 export const llamaBinsGrammarsDirectory = path.join(__dirname, "..", "llama", "grammars");
 export const projectTemplatesDirectory = path.join(__dirname, "..", "templates");
-export const localTempDirectory = path.join(__dirname, "..", ".temp");
+export const localTempDirectory = path.join(nixCacheDir, ".temp");
 export const packedProjectTemplatesDirectory = path.join(projectTemplatesDirectory, "packed");
-export const llamaCppDirectory = path.join(llamaDirectory, "llama.cpp");
+export const llamaCppDirectory = path.join(nixCacheDir, "llama.cpp");
 export const llamaCppPatchesDirectory = path.join(llamaDirectory, "patches");
-export const llamaCppGrammarsDirectory = path.join(llamaDirectory, "llama.cpp", "grammars");
+export const llamaCppGrammarsDirectory = path.join(nixCacheDir, "llama.cpp", "grammars");
 export const tempDownloadDirectory = path.join(os.tmpdir(), "node-llama-cpp", nanoid());
 export const cliHomedirDirectory = path.join(os.homedir(), ".node-llama-cpp");
 export const cliHomedirTempDirectory = path.join(os.homedir(), ".node-llama-cpp", ".temp");
 export const chatCommandHistoryFilePath = path.join(cliHomedirDirectory, ".chat_repl_history");
 export const cliModelsDirectory = path.join(cliHomedirDirectory, "models");
-export const lastBuildInfoJsonPath = path.join(llamaDirectory, "lastBuild.json");
+export const lastBuildInfoJsonPath = path.join(nixCacheDir, "lastBuild.json");
 export const binariesGithubReleasePath = path.join(llamaDirectory, "binariesGithubRelease.json");
-export const llamaCppDirectoryInfoFilePath = path.join(llamaDirectory, "llama.cpp.info.json");
+export const llamaCppDirectoryInfoFilePath = path.join(nixCacheDir, "llama.cpp.info.json");
 export const currentReleaseGitBundlePath = path.join(llamaDirectory, "gitRelease.bundle");
-export const xpackDirectory = path.join(llamaDirectory, "xpack");
+export const xpackDirectory = path.join(nixCacheDir, "xpack");
 export const localXpacksStoreDirectory = path.join(xpackDirectory, "store");
 export const localXpacksCacheDirectory = path.join(xpackDirectory, "cache");
 export const buildMetadataFileName = "_nlcBuildMetadata.json";
